src/persist-client/src/usage.rs

```diff
@@ -140,5 +170,17 @@ mod tests {
 
         assert_eq!(usage.shard_size(&shard_id_one).await, shard_one_size);
         assert_eq!(usage.shard_size(&shard_id_two).await, shard_two_size);
+
+        let (shard_sizes, unattributable) = usage.shard_sizes().await;
+        assert_eq!(unattributable, 0);
+        assert_eq!(shard_sizes.len(), 2);
+        assert_eq!(
+            *shard_sizes.get(&shard_id_one).expect("must have shard 1"),
```

Comment:
nit: instead of the * and the expect, I'd assert_eq against `Some(&shard_one_size)`

Reply:
oh yeah that reads a lot better. I feel like I don't fully understand references to Copy types in Rust